src/app/dataset-flow/dataset-flow-details/tabs/flow-details-history-tab/flow-details-history-tab.helpers.ts

```diff
@@ -64,6 +67,8 @@ export class DatasetFlowDetailsHelpers {
                 return { icon: "downloading", class: "text-muted" };
             case "FlowEventScheduledForActivation":
                 return { icon: "timer", class: "text-muted" };
+            case "FlowConfigSnapshotModified":
+                return { icon: "downloading", class: "text-muted" };
```

Comment:
I think `FlowEventStartConditionUpdated` and `FlowConfigSnapshotModified` should have different icons. Start condition represents state change of the flow while it's waiting (we are waiting for schedule, throttling, executor, ...), and config snapshot represents the changes have been made to the configuration.

Reply:
changed icon